docs/metrics.md

```diff
@@ -0,0 +1,175 @@
+
+https://github.com/mozilla-mobile/focus-android/wiki/Event-Tracking-with-Mozilla%27s-Telemetry-Service
+
+TODO: link to the lockbox for android source code at the point where the pings and event values are defined (when that code exists).
+
+See [this](https://github.com/mozilla-mobile/focus-android/blob/master/app/src/main/java/org/mozilla/focus/telemetry/TelemetryWrapper.kt) for the kotlin source code that Firefox Focus uses to define its telemetry events.
+
+Every event must contain `category`, `method` and `object` fields, and may optionally contain `value` and `extra` fields as well.
+
+Events related to specific credentials should have an opaque `item_id` in the extra field where possible.
+
+
+Finally, the `appName` metadata sent with each ping should always be 'Lockbox'.
+
+See here for more information on event schemas:
+
+https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/telemetry/collection/events.html#public-js-api
+
+## List of Proposed Events
+
+1. When the app starts up:
+	* `category`: action
+	* `method`: startup
+	* `object`: app
+	* `value`: null
+	* `extras`: null
+
+2. When locking/unlocking the app:
+	* `category`: action
+	* `method`: lock, unlock
+	* `object`: app
+	* `value`: pin, biometrics
+	* `extras`: null
+
+3. Events that fire during the setup process:
+	* `category`: action
+	* `method`: show
+	* `object`: login_welcome, login_fxa, login_learn_more, biometrics_setup, autofill_setup
+	* `value`: null
+	* `extras`: null
+
+4. When the main item list is shown to the user:
+	* `category`: action
+	* `method`: show
+	* `object`: entryList
+	* `value`: null
+	* `extras`: null
+
+5. When a user shows the details of an item in the entry list:
+	* `category`: action
+	* `method`: show
+	* `object`: entryDetail
+	* `value`: null
+	* `extras`: ["itemid" : itemid]
+
+6. When a user taps one of the copy buttons available after being shown entry details:
+	* `category`: action
+	* `method`: tap
+	* `object`: entryCopyUsernameButton, entryCopyPasswordButton
+	* `value`: null
+	* `extras`: ["itemid" : itemid]
+
+7. When a user shows details from an item, is the password shown?:
+	* `category`: action
+	* `method`: tap
+	* `object`: reveal_password
+	* `value`: true or false, whether the pw is displayed
+	* `extras`: null
+
+8. When one of the settings pages is shown to the user:
+	* `category`: action
+	* `method`: show
+	* `object`: settings_list, settings_autolock, settings_preferred_browser, settings_account, settings_faq, settings_provide_feedback, settings_autofill
+	* `value`: null
+	* `extras`: null
+
+9. When a user changes something on the settings page:
+	* `category`: action
+	* `method`: settingsChanged
+	* `object`: settings_biometric_login, settings_autolock_time, settings_reset, settings_visual_lock, settings_preferred_browser, settings_record_usage_data
```

Comment:
Discussed some more in IRL:

* remove events for `settings_preferred_browser`, `settings_record_usage_data`
* determine what `settings_visual_lock` maps to (or remove)